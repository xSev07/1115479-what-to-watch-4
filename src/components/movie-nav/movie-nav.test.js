import React from "react";
import renderer from "react-test-renderer";
import MovieNav from "./movie-nav";

it(`Should MovieNav render correctly`, () => {
  const tree = renderer
    .create(
        <MovieNav
          tabs={[`tab1`, `tab2`, `tab3`]}
          activeTab={`tab2`}
          onClick={() => {}}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
