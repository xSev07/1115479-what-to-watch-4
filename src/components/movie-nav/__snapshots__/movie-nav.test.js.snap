// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieNav render correctly 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item"
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        tab1
      </a>
    </li>
    <li
      className="movie-nav__item movie-nav__item--active"
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        tab2
      </a>
    </li>
    <li
      className="movie-nav__item"
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        tab3
      </a>
    </li>
  </ul>
</nav>
`;
